packages/lib-classifier/dev/components/App/App.js

```diff
@@ -79,20 +80,24 @@ class App extends React.Component {
       )
     }
 
+    const mergedThemes = _.merge({}, baseTheme, zooTheme)
+
     return (
-      <Grommet theme={theme}>
-        <Box tag='header' pad='medium' align='end'>
-          {this.state.user
-            ? <Button onClick={this.logout.bind(this)} label='Logout' />
-            : <Button onClick={this.login.bind(this)} label='Login' />
-          }
-        </Box>
-        <Box tag='section'>
-          <Classifier
-            authClient={oauth}
-            onCompleteClassification={(classification, subject) => console.log('onComplete', classification, subject)}
-            project={this.state.project}
-          />
+      <Grommet theme={mergedThemes}>
+        <Box as='main' background={zooTheme.global.colors['light-1']}>
+          <Box as='header' pad='medium' align='end'>
+            {this.state.user
+              ? <Button onClick={this.logout.bind(this)} label='Logout' />
```

Comment:
I know this isn't one of your changes, but this should be done in the constructor